app/models/game.rb

```diff
@@ -5,4 +5,128 @@ class Game < ActiveRecord::Base
   belongs_to :white_user, class_name: 'User', foreign_key: 'white_uid'
   belongs_to :black_user, class_name: 'User', foreign_key: 'black_uid'
 
+  after_create :populate_board!
+  
+ 	def populate_board!
+ 		game = Game.last.id
+	
+		######create rooks######
+		0.step(7,7) do |x|
+			Piece.create!(
```

Comment:
each piece should be responsible for initializing itself.  The populate board should be concerned with creating new pieces associated with the proper user and the proper game, but the initial position should be encapsulated in the pieces class.  Additionally you should not be creating a "Piece" but create a "Rook"...  Try re-doing this, this time using polymorphism to initialize the pieces.